src/organisation/components/pba-numbers-form/pba-numbers-form.component.html

```diff
@@ -5,9 +5,9 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
 
-        <table class="govuk-table" *ngIf="updatePbaNumbers.hasDisplayPbaNumbers; else noPbaNumbersMessage">
+        <table class="govuk-table" *ngIf="getCurrentPaymentAccounts().length; else noPbaNumbersMessage">
```

Comment:
I am not sure an AsyncPipe is the best for this because there is nothing really asynchronous about the processing, i.e. we are not showing anything that is processed by the store. I have however gotten rid of the extraneous If statement and moved the current PBA get into a get operator and called from there instead and taken more logic out of the template around this area